### ISSUE TITLE
Can't get the example to work

### ISSUE THREAD

**Issue by SebastianBO** [EurekaCommunity/VideoRow#2]
Showing Recent Messages
:-1: /Users///VideoRow/Example/Pods/Pods/Target Support Files/Pods-VideoRow_Tests/Pods-VideoRow_Tests.debug.xcconfig: unable to open file (in target "VideoRow_Tests" in project "VideoRow") (in target 'VideoRow_Tests')

Error when running

**Comment by SebastianBO** [comment]
Also when looking at code without git clone -> pod install -> run it seems to not have anything inside of it.  
VideoRow contains

import Foundation
public class VideoRow {
    var pointlessProperty: Any
    
    public init(pointlessParam: Any) {
        self.pointlessProperty = pointlessParam
    }
    
    public func temp() {
        print("this prints to the console so we can see if this is working!")
    }
}

**Comment by Smiller193** [comment]
@SebastianBO yeah its not ready yet bro this is my first time doing this just be a little patient

**Comment by Smiller193** [comment]
@SebastianBO try now

**Comment by SebastianBO** [comment]
Pod still not working for me, but I'll try to make it work! Thanks for the update :)

**Comment by Smiller193** [comment]
Did you update the pod ?

**Comment by SebastianBO** [comment]
Done it all, re-clone / pod update etc.

Versions:
Installing Eureka (4.3.0)
Installing TLPhotoPicker (1.7.7)
Installing VideoRow (0.1.1)

**Comment by Smiller193** [comment]
Whats the error

**Comment by Smiller193** [comment]
@mats-claassen can you test it to see if it is not working for you as well

**Comment by Smiller193** [comment]
@SebastianBO I just tested it again created a whole new project with a new pod and installed it and it worked for me

**Comment by mats-claassen** [comment]
I could run the Example on the simulator but not on the device. I have some trouble with the Pods xcconfig files which seem to have a wrong path.

Adding the pod to another project works, BTW.

I just found that `_VideoRow` is the final class while `VideoRow` is the generic superclass. We normally do this the other way around as it is nicer to use `VideoRow` than `_VideoRow` in our projects. I would suggest to swap those names.

**Comment by Smiller193** [comment]
@mats-claassen im going to alter that and test it on a device now

**Comment by Smiller193** [comment]
@mats-claassen test one more time

**Comment by mats-claassen** [comment]
I am still getting the duplicated xcconfig files issue but I am not sure what is causing that. Cloned the repo, then ran `pod install` in the `Example` folder.

![screen shot 2018-11-14 at 10 02 07 am](https://user-images.githubusercontent.com/8573865/48484306-8d180c00-e7f4-11e8-820c-21126bd592a1.png)


The duplicate red files have a wrong and non-existent path set

**Comment by Smiller193** [comment]
Ummm I’ll look into how to fix that

**Comment by Smiller193** [comment]
@mats-claassen havent been able to locate a fix. Is it causing the pod to not function properly?

**Comment by Smiller193** [comment]
@SebastianBO can you still not get the example to work?

**Comment by mats-claassen** [comment]
The pod worked properly for me. It is just an example issue
